sensing/src/main/java/com/google/android/sensing/impl/SensorManagerImpl.kt

```diff
@@ -227,6 +236,40 @@ internal class SensorManagerImpl(context: Context, private val database: Databas
     }
   }
 
+  /**
+   * Zips the folder data + metadata were captured to and add an UploadRequest in the database. TODO
```

Comment:
is the documentation accurate? or maybe this function needs a better name?

Reply:
also don't use `+`. sounds too casual.

Reply:
actually this documentation makes me think this function needs to be split into two.